.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @infratographer/the_committee @infratographer/load_balancer_maintainers
```

Comment:
@infratographer/load_balancer_maintainers doesn't have write access to this repo